test/application_system_test_case.rb

```diff
@@ -0,0 +1,5 @@
+require "test_helper"
```

Comment:
Let's skip tests also so we use the `rspec` library instead. You can simply delete the `test` folder or use the flag `-T` when creating the app.
```ruby
rails new ... -T
```